.github/workflows/action-install-from-conda.yml

```diff
@@ -34,7 +35,9 @@ jobs:
       - shell: bash -l {0}
         run: python -V 2>&1 | tee conda_install_linux_artifacts_python_${{ matrix.python-version }}/python_version.txt
       - shell: bash -l {0}
-        run: conda install esmvaltool 2>&1 | tee conda_install_linux_artifacts_python_${{ matrix.python-version }}/install.txt
+        run: conda install mamba
```

Comment:
Mamba suggests to install mamba in `-n base`. Do you want to avoid that on purpose?

Reply:
no but the environment gets created at the top of the Github Action job, exiting it to stay in `base` then re-entering it is a bit sillyo

Reply:
> I just noticed that the order of the channels here is esmvalgroup,conda-forge. Could that mean that this picks up the esmvalcore from our own channel (which has cython as a dependency itself), instead of the conda-forge one?

indeed it could be that but not 100% sure since we're not setting channel priority to strict, but now you've fixed it on conda-forge so it should be all the same

Reply:
> no but the environment gets created at the top of the Github Action job, exiting it to stay in `base` then re-entering it is a bit sillyo

I don't think you need to do that. Just add `-n base` to the command line.